app/commands.py

```diff
@@ -961,3 +962,44 @@ def pprint(msg):
     pprint("Committing...")
     db.session.commit()
     pprint("Finished.")
+
+
+@notify_command(name="purge-alerts")
+@click.option(
+    "-o",
+    "--older-than",
+    required=False,
+    type=int,
+    help="""Alerts older than the provided value (in days) will be purged from the database""",
+)
+@click.option(
+    "-s",
+    "--service",
+    required=False,
+    default=None,
+    help="""Service identifier - can be either the service UUID or the service name""",
+)
+@click.option(
+    "-d",
+    "--dry-run",
+    required=False,
+    default=False,
+    type=bool,
+    help="""Show the IDs of the DB items selected only. The items will not be deleted""",
+)
+def purge_alerts_from_db(older_than, service, dry_run):
+    if os.environ.get("ENVIRONMENT") != "preview":
```

Comment:
@jonathan-owens-gds  Just wanted to confirm, the purging of items is only allowed in preview/dev, as printed on the next line. Shouldn't this condition therefore include dev? Doesn't really impact anything, just for consistency

Reply:
I've modified this to be consistent